PyInstaller/hooks/hook-pydantic.py

```diff
@@ -0,0 +1,35 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2005-2020, PyInstaller Development Team.
+#
+# Distributed under the terms of the GNU General Public License (version 2
+# or later) with exception for distributing the bootloader.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#
+# SPDX-License-Identifier: (GPL-2.0-or-later WITH Bootloader-exception)
+#-----------------------------------------------------------------------------
+
+# pydatic ....
+
+hiddenimports = [
+    'pydantic.class_validators',
+    'pydantic.color', 'colorsys',
+    'pydantic.dataclasses',
+    'pydantic.datetime_parse',
+    'pydantic.decorator',
+    'pydantic.env_settings',
+    'pydantic.error_wrappers',
+    'pydantic.errors',
+    'pydantic.fields',
+    'pydantic.json', 'json', 'ipaddress', 'uuid',
+    'pydantic.main',
+    'pydantic.networks',
+    'pydantic.parse',
+    'pydantic.schema',
+    'pydantic.tools',
+    'pydantic.types',
+    'pydantic.typing',
+    'pydantic.utils',
+    'pydantic.validators',
+    'pydantic.version',
+]
```

Comment:
Any reason we can't use `collect_submodules`?

Reply:
All these are extension modules, too. I did not test whether collect_submodules would work. Anyway, this hook should go into hooks-contrib.

Reply:
@htgoebel we can move it later. It's needed for the CI to run so it'll have to stay in PyInstaller for now.

Reply:
Where is pydandic used in CI?
```
$ git grep pydantic
$
```

Reply:
Oh, it's not. Would you mind sticking this in a PR to hooks contrib? After that this is ready to merge - thanks for fixing this.

Reply:
> sticking this in a PR to hooks contrib

yes, will take a few days until I find time for this, though.

Reply:
See https://github.com/pyinstaller/pyinstaller-hooks-contrib/pull/57